app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Indicate a list of what instances you expect to obtain for `friends` in the type annotation